nni/contrib/compression/base/wrapper.py

```diff
@@ -75,6 +87,33 @@ def __init__(self, module: torch.nn.Module, module_name: str, config: Dict[str,
         # An example please refer ``track_target_shape``.
         self._track_funcs: List[Callable[[ModuleWrapper, str, Tensor], None]] = [track_target_shape]
 
+        # register for module fusion
+        self.register_fusion_info(fused_modules)
+
+    def register_fusion_info(self, fused_modules: List[torch.nn.Module] | None = None):
+        self.fused_modules = fused_modules if fused_modules is not None else []
+        if len(self.fused_modules) > 0:
+            self.is_bias = check_bias(self.module) # used for fold_bn
+            self.register_bias()
```

Comment:
call this func in `wrap()`? we don't wrap the module at this time

Reply:
okok

Reply:
Consider a case, if a module has wrapped in the pruning compressor, then it can not be wrapped for module fusion.
WARNING: Wrapper of module is wrapped, no need to wrap again.